### ISSUE TITLE
Runtime path is not portable by default - fails if templates are packaged/reused

### ISSUE THREAD

**Issue by rosskevin** [emaphp/handlebars-template-loader#27]
If the templates are compiled into a package called `mytemplate-library` by a CI server, then used on another machine in any other environment, template loading will fail. 

## default config
Specifically, by default:

```js
      {
        exclude: /node_modules/,
        loader: 'handlebars-template-loader',
        test: /\.hbs/,
      },
```

```js
runtimePath = require.resolve('handlebars/runtime').replace(/\\/g, '/')
```

resolves the runtime to a fully absolute hardcoded path on the build machine e.g. `/home/circleci/...` which doesn't exist elsewhere and is not using node module resolution.

## manually specify runtime config
When I set:

```js
      {
        exclude: /node_modules/,
        loader: 'handlebars-template-loader',
        test: /\.hbs/,
        query: {
          runtimePath: 'handlebars/runtime',
        },
      },
```

the generated code is then trying to load from `require('./dist/cjs/handlebars.runtime')` which is nowhere to be found in `mytemplate-library` package, but it is in the `node_modules/handlebars/runtime.js`.

I'm confused as to why this is not generating a string `var Handlebars = require('handlebars/runtime');`, webpack seems to be looking in and resolving the alias at compile time, when I need it to resolve this at runtime instead.  In other words, the `node_modules/handlebars` package exists at runtime, but at compile time, this loader is generating  incorrect paths - it simply needs to not resolve the code until runtime.

## TLDR
The code generated by this loader does not appear to be usable in default or custom runtime-specified form when compiled into an intermediate library package.

Any ideas on solving this?

**Comment by rosskevin** [comment]
I just spotted a clue - `handlebars/runtime` was not in `externals`, trying that now to see what happens.

**Comment by rosskevin** [comment]
Confirmed, it was an error in the webpack configuration of externals.
